### PR TITLE
fix Jest testRegex param to match neutrino test dir

### PR DESCRIPTION
**Pull request by jefo**
Quickfix for #518.

### CODE REVIEW

File: packages/jest/src/index.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ function normalizeJestOptions(opts, neutrino, usingBabel) {
       bail: true,
       coveragePathIgnorePatterns: [neutrino.options.node_modules],
       collectCoverageFrom: [join(basename(neutrino.options.source), '**/*.js')],
```
Comment (eliperelman) [inline packages/jest/src/index.js]: @helfi92 looks like there are a few outstanding extensions here.

File: packages/jest/src/index.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ function normalizeJestOptions(opts, neutrino, usingBabel) {
...
-      testRegex: join(basename(neutrino.options.tests), '.*(_test|_spec|\\.test|\\.spec)\\.jsx?$'),
+      testRegex: join(basename(neutrino.options.tests), '.*(_test|_spec|\\.test|\\.spec)\\.jsx?$').replace('test\\', '(test\\)'),
```
Comment (eliperelman) [inline packages/jest/src/index.js]: @jefo if you log the value of `testRegex` before and after this change, what does it log?

**Comment by eliperelman** [comment]
Adding @okonet as he may have some insight here. 😃

**Comment by sgillespie** [comment]
I do not think this is necessary anymore, since #679 has been merged

**Comment by edmorley** [comment]
> I do not think this is necessary anymore, since #679 has been merged

Ah ok. Thank you for the PR anyway, much appreciated :-)

**Pull request closed without merging**